cmd/juju/model/exportbundle.go

```diff
@@ -55,7 +54,7 @@ func (c *exportBundleCommand) Info() *cmd.Info {
 // SetFlags implements Command.
 func (c *exportBundleCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.ModelCommandBase.SetFlags(f)
-	c.out.AddFlags(f, "yaml", output.DefaultFormatters)
+	//c.out.AddFlags(f, "yaml", output.DefaultFormatters)
```

Comment:
Delete this all together if not needed?...

Reply:
we dont need to have --format and --output options. AddFLags just injects both cli flags. So i removed.

Reply:
You'll need to modify the body of the run method too most likely...

Reply:
My original comment was to point out that it is not good enough to comment this code out. Remove it! 
We don not want to clatter our production code with commented out code :D